poonam/JS/Assignment12_js/que2.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>que 2</title>
+    </head>
+    <body>
+        <script>
+            class Employee {
+                constructor(empno, name, address, city, salary) {
```

Comment:
Pass empty constructor, Because value come from **getData() Method**
```suggestion
                constructor() {
```